spec/features/two_factor_authentication/backup_code_sign_up_spec.rb

```diff
@@ -22,14 +22,14 @@
     expect(user.backup_code_configurations.count).to eq(10)
   end
 
-  it 'does not show download button on a mobile device' do
+  it 'does show download button on a mobile device' do
     allow(BrowserCache).to receive(:parse).and_return(mobile_device)
 
     sign_up_and_set_password
 
     select_2fa_option('backup_code')
 
-    expect(page).to_not have_link(t('components.download_button.label'))
+    expect(page).to have_link(t('components.download_button.label'))
   end
```

Comment:
Maybe we don't need this spec at all, if there's nothing in the view which we expect to differ now based on mobile vs. desktop, since we have an equivalent assertion in the "allows backup code only MFA configurations" above.